RelativityDev.RelativityTestHelpers.nuspec

```diff
@@ -1,29 +1,32 @@
-<?xml version="1.0"?>
-<package >
-  <metadata>
-    <id>RelativityDev.RelativityTestHelpers</id>
-    <version>6.0.0.1</version>
-    <authors>RelativityDev</authors>
-    <owners>RelativityDev</owners>
-    <licenseUrl>https://github.com/relativitydev/relativity-test-helpers/blob/master/LICENSE</licenseUrl>
-    <projectUrl>https://github.com/relativitydev/relativity-test-helpers</projectUrl>
-    <iconUrl>https://bit.ly/2qFJQzk</iconUrl>
-    <requireLicenseAcceptance>true</requireLicenseAcceptance>
-    <description>This NuGet package contains a helper library to write integration tests for testing Relativity applications. This is a community supported implementation. If this does not meet your needs for your implementation you are invited to contribute to the project.</description>
-    <releaseNotes>https://github.com/relativitydev/relativity-test-helpers</releaseNotes>
-    <copyright></copyright>
-    <tags>relativity-test-helpers relativity test helpers relativity test helpers test helpers integration tests relativity integration tests </tags>
-	<dependencies>
-      <group targetFramework=".NETFramework4.6.2">
-        <dependency id="Relativity.DataExchange.Client.SDK" version="1.4.0" />
-        <dependency id="Relativity.ObjectManager" version="10.3.170.1" />
-        <dependency id="Newtonsoft.Json" version="12.0.2" />
-        <dependency id="MailKit" version="2.2.0" />
-        <dependency id="MimeKit" version="2.2.0" />
-      </group>
-    </dependencies>
-  </metadata>
-  <files>
-    <file src="Source\Relativity.Test.Helpers\Relativity.Test.Helpers\bin\$configuration$\Relativity.Test.Helpers.dll" target="lib\net462\" />
-  </files>
-</package>
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>RelativityDev.RelativityTestHelpers</id>
+    <version>6.0.0.1</version>

```

Comment:
this still says version 6

Reply:
i don't think it's an issue and pipeline will probably overwrite it with the version number specified in the pipeline variable.